Sources/TuistGenerator/Linter/TargetLinter.swift

```diff
@@ -150,15 +152,17 @@ class TargetLinter: TargetLinting {
     }
 
     private func lintInfoplistExists(target: Target) -> [LintingIssue] {
-        var issues: [LintingIssue] = []
-        if let infoPlist = target.infoPlist,
-           case let InfoPlist.file(path: path) = infoPlist,
-           !FileHandler.shared.exists(path)
-        {
-            issues
-                .append(LintingIssue(reason: "Info.plist file not found at path \(infoPlist.path!.pathString)", severity: .error))
-        }
-        return issues
+        var infoPlists = target.settings?.configurations.infoPlists() ?? []
+        target.infoPlist.map { infoPlists.append($0) }
```

Comment:
```suggestion
        let infoPlists = [target.infoPlist] + target.settings?.configurations.infoPlists() ?? []
```

Reply:
I don't think this will work because target.infoPlist is Optional and type of `[target.infoPlist]` will be `[InfoPlist?]` and it can't be concatenated with `[InfoPlist]`